src/app/profile/settings/connected-accounts/connected-accounts.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="container-fluid container-cards-pf">
+  <div class="row-cards-pf accounts-component">
+    <div class="col-xs-10 col-sm-4 col-md-4">
```

Comment:
You can remove the additional `col-md-4`, as `col-sm-4` defines the same column width.